docs/explanation/generic-rules.md

```diff
@@ -0,0 +1,68 @@
+# Generic alert rule groups
+The Canonical Observability Stack comes with Generic alerts rules that have been introduced to provide a minimal set of rules such that they inform admins when hosts in a deployment are unhealthy, unreachable, or otherwise unresponsive. This helps alleviate charm authors from having to implement their host-health-related alerts per charm. 
+There are two generic alert rule groups: `HostHealth` and `AggregatorHostHealth`, each containing multiple alert rules.
+This guide provides explanations on the purposes of each rule group and its alerts. For steps to troubleshoot firing alert rules, please refer to the [troubleshooting guide](../how-to/troubleshooting/troubleshoot-firing-alert-rules.md).
+
+The `HostHealth` and `AggregatorHostHealth` alert rule groups are applicable to the following deployment scenarios:
+
+```{mermaid}
+graph LR
+
+subgraph lxd
+vm-charm1 ---|cos_agent| grafana-agent
+vm-charm2 ---|monitors| cos-proxy
+end
+
+subgraph k8s
+k8s-charm1 ---|metrics_endpoint| prometheus
+k8s-charm2 ---|metrics_endpoint| grafana-agent-k8s
+grafana-agent-k8s ---|prometheus_remote_write| prometheus
+end
+
+grafana-agent ---|prometheus_remote_write| prometheus
+cos-proxy ---|metrics_endpoint| prometheus
+```
+You can find more information on these groups and the alert rules they contain below.
+
+## `HostHealth` alert group
+The `HostHealth` alert rule group contains the `HostDown` and `HostMetricsMissing` alert rules, identifying unreachable target scenarios.
+
+### `HostDown` alert rule
+The purpose of this alert is to notify when Prometheus (or Mimir) failed to scrape its target. The alert expression executes `up{...} < 1` with labels including the target's Juju topology: `juju_model`, `juju_application`, etc. In essence, this alert notifies you in case of failed scrapes and is directly applicable to cases where a charm is being scraped by Prometheus for metrics. The [`up` metric](https://prometheus.io/docs/concepts/jobs_instances/), which is what this alert's expression relies on, indicates the health or reachability status of a node. For example, when `up` is 1 for a charm, this is a sign that Prometheus is able to successfully call the metrics endpoint of that charm and access the metrics that are exposed at that endpoint.
+
+This alert is especially important for Cos Lite, where Prometheus is capable of scraping charms for metrics. The firing of this alert indicates that Prometheus is not able to scrape a target for metrics, leading to `up` being 0.
+
+
+### `HostMetricsMissing`
+```{note}
+`HostMetricsMissing` is also used in the `AggregatorHostHealth` group. As part of the `HostHealth` group, however, it monitors the health of all charms whose metrics are collected by an aggregator and then remote written to a metrics backend and not just aggregators (see the `AggregatorHostHealth` group for details on the distinction).
```

Comment:
```suggestion
`HostMetricsMissing` is also used in the `AggregatorHostHealth` group. As part of the `HostHealth` group, however, it monitors the health of any charm (not just aggregators) whose metrics are collected by an aggregator and then remote written to a metrics backend. See the `AggregatorHostHealth` group for details on the distinction.
```
This note was confusing to me. Moving the clarifying part earlier in the paragraph makes it better. Although feel free to make changes to the language if my suggestion changed the meaning of it